CMakeLists.txt

```
@@ -32,9 +32,9 @@ option(HIP_DNN_BUILD_FRONTEND "Build the frontend code" ON)
 option(HIP_DNN_SKIP_TESTS "Skips building all tests" OFF)
 option(HIP_DNN_BUILD_PLUGINS "Builds the plugins as part of the main cmake" OFF)
 
-
 option(CODE_COVERAGE "Build with code coverage flags" OFF)
 option(ENABLE_CLANG_TIDY "Enable Clang-Tidy checks" ON)
+option(BUILD_ADDRESS_SANITIZER "Build with Address Sanitizer enabled" OFF)
```

[R] I'm not insisting, but I think this way would be more informative.
```suggestion
option(ENABLE_ASAN "Build with Address Sanitizer enabled" OFF)
```

---

This came from a conversation with Eiden.  I think they're trying to co-ordinate build options a bit amongst packages.